README.md

```diff
@@ -15,8 +15,10 @@ __Note__: DataStax products do not support big-endian systems.
 ## Getting the Driver
 
 Binary versions of the driver, available for multiple operating systems and
-multiple versions of PHP, can be obtained from our [download server]. The
-source code is made available via [GitHub]. __If using [DataStax Enterprise]
+multiple versions of PHP, can be obtained from [DataStax download server] (binary
+packages may have dependency on packages of [DataStax C/C++ Driver for Apache Cassandra] and
+its dependencies, so you may need to install them as well). The
```

Comment:
The new additions in `()`'s feels a little breathy; what do you think about breaking it up into a __Note__ like the following:

```
Binary versions of the driver, available for multiple operating systems and
multiple versions of PHP, can be obtained from [DataStax download server].
The source code is made available via [GitHub]. __If you're using
[DataStax Enterprise] use the [DSE PHP driver] instead__.

__Note__: The driver extension is a wrapper around the
          [DataStax C/C++ Driver for Apache Cassandra] and is a requirement for
          proper installation of the PHP extension binaries. Ensure these
          dependencies are met before proceeding.
```